erpnext/support/doctype/issue/issue.json

```diff
@@ -362,12 +370,57 @@
    "label": "Issue Split From",
    "options": "Issue",
    "read_only": 1
+  },
+  {
+   "fieldname": "column_break_26",
+   "fieldtype": "Column Break"
+  },
+  {
+   "bold": 1,
+   "fieldname": "avg_response_time",
```

Comment:
Rather than showing an average response time per issue, why not create a report directly ??

Reply:
Yes, we can have a report too, but the idea of having the metric inside the issue is to analyze how was the response time for a particular issue has been so that if there are complaints regarding slower responses from a customer, we can see it clearly in a ticket.